lib/CUDD/CMakeLists.txt

```diff
@@ -1,20 +1,20 @@
 set(SOURCES
        cuddAddNeg.c
        cuddBddCorr.c
-       cuddDecomp.c
+       cuddDecomp.cpp
```

Comment:
could you show the error that you're getting? the travis mac builds (and my mac builds) are showing clean.

are you running a clean build (a fresh git clone)?